src/providers/export/cntk.ts

```diff
@@ -3,6 +3,7 @@ import { ExportProvider, IExportResults } from "./exportProvider";
 import { IAssetMetadata, IExportProviderOptions, IProject } from "../../models/applicationState";
 import HtmlFileReader from "../../common/htmlFileReader";
 import Guard from "../../common/guard";
+import {splitTestAsset} from "./testAssetsSplitHelper";
```

Comment:
Nit: Spacing around `{` in import statements.

Reply:
Fixed in bfce1ec